frameworks/CSharp/aspnetcore/PlatformBenchmarks/appsettings.postgresql.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "ConnectionString": "Server=tfb-database;Database=hello_world;User Id=benchmarkdbuser;Password=benchmarkdbpass;Maximum Pool Size=256;NoResetOnClose=true;Enlist=false;Max Auto Prepare=4;Multiplexing=true;Write Coalescing Delay Us=500;Write Coalescing Buffer Threshold Bytes=1000",
```

Comment:
@roji is this the correct connection string for Platform? I've kept updates the same (one below)

Reply:
Yeah that's the good one for multiplexing (though have we turned that on for all database scenarios, or just fortunes?)

Reply:
Would be turned on for fortunes, single query and multi-query (but not updates). Should they use different ones?

Reply:
Sorry for not answering this, have been underwater. I plan a thorough pass comparing 5.0 multiplexing, 5.0 non-multiplexing and 4.1.3 perf (i.e. currentl released version of Npgsql) for platform and non-platform benchmarks... I should be able to look into this in a couple of weeks.